grammar.js

```diff
@@ -1093,7 +1093,7 @@ module.exports = grammar({
     ),
 
     anonymous_method_expression: $ => seq(
-      optional('async'),
+      optional(alias(choice('async', 'static', seq('async', 'static'), seq('static', 'async')), $.modifier)),
```

Comment:
Why doesn't `repeat($.modifier)` work here?

Reply:
We've had a lot of problems with modifiers - they are so broad that in many places it causes conflicts while allowing invalid syntax for specific scenarios (which I think Roslyn flags in a second-stage)